spyder/widgets/panels/__init__.py

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Please add our header here too and a docstring describing what this is the purpose of this module.